t/plugin/request-validation.t

```diff
@@ -358,3 +358,33 @@ required field missing
 property "required_payload" is required
 
 
+
+=== TEST 8: required header added in header
+--- config
+    location /t {
+        content_by_lua_block {
+            local http = require "resty.http"
+            local httpc = http.new()
+            local uri = "http://127.0.0.1:" .. ngx.var.server_port .. "/opentracing"
+            local res, err = httpc:request_uri(uri,
+                {
+                    method = "GET",
+                    headers = {
+                        ["Content-Type"] = "application/json",
+                        ["required_payload"] = "test payload"
+                    }
+                })
+
+            if res.status == 200 then
+                ngx.say("hello1 world")
+            else
+                ngx.say("failed")
+            end
+        }
+    }
+--- request
+GET /t
+--- response_body
+hello1 world
+--- no_error_log
+[error]
```

Comment:
need one blank line at the end of the file.

![image](https://user-images.githubusercontent.com/6814606/87761457-12fb1e80-c844-11ea-84f4-c8e23e433936.png)